public_src/components/toolbar/toolbar.component.html

```diff
@@ -16,6 +16,17 @@
 
 <transporter></transporter>
 
+
+<button class="btn btn-primary on-map route-labels-btn" *ngIf="enableInfoRouteLabels" [disabled]="storageSrv.currentElement.type === 'relation' && !hasRef()"
```

Comment:
maybe these labels should be enabled by default, it took time to figure out how to enable it but it always resets to disabled when I click or hover over any other route in the sidebar list.